src/main/java/com/epam/aidial/core/controller/DeploymentPostController.java

```diff
@@ -163,12 +163,14 @@ private void handleProxyRequest(HttpClientRequest proxyRequest) {
         if (!deployment.isForwardApiKey()) {
             excludeHeaders.add(Proxy.HEADER_API_KEY);
         }
-        if (!deployment.isForwardAuthToken()) {
-            excludeHeaders.add(HttpHeaders.AUTHORIZATION);
-        }
+        request.headers().remove(Proxy.HEADER_API_KEY);
```

Comment:
Another thing that worries me, we have "project" stored everywhere in logs based on key used. See: GfLogStore for instance as well as controllers code. What are we going to store for requests that uses auto generated keys? 

Ideally it would be nice to store model/app/assistant name.

Reply:
fixed